src/tb/renderers/tb_renderer_gl.cpp

```diff
@@ -178,7 +178,7 @@ TBRendererGL::TBRendererGL()
 	m_program = glCreateProgram();
 	if (m_program == 0)
 	{
-		TBDebugPrint("glCreateProgram failed.\n");
+		TBDebugPrint("glCreateProgram failed.\n", 0);
```

Comment:
Use `TBDebugOut` if there are no parameters. `TBDebugPrint` is a macro for printing with parameters that calls `TBDebugOut` internally.